src/components/Autosuggest/Autosuggest.styles.js

```diff
@@ -75,7 +75,8 @@ const styles = {
       marginBottom: Space.S0,
       maxHeight: '250px',
       overflow: 'scroll',
-      '-webkit-overflow-scrolling': 'touch',
+      overflowScrolling: 'touch',
```

Comment:
Last nit 😬: this can be removed, overflow scrolling only exists under the `-wekbit` prefix.